Terminal.Gui/Application/Application.cs

```diff
@@ -708,7 +714,10 @@ public static T Run<T> (Func<Exception, bool> errorHandler = null, ConsoleDriver
     ///         <see cref="RunLoop(RunState)"/> method will only process any pending events, timers, idle handlers and then
     ///         return control immediately.
     ///     </para>
-    ///     <para>Calling <see cref="Init"/> first is not needed as this function will initialize the application.</para>
+    ///     <para>Calling <see cref="Init"/> first is not needed if only <see cref="Run{T}"/> or
```

Comment:
///     <para>When using <see cref="Run{T}"/> or
    ///         <see cref="Run(System.Func{System.Exception,bool},Terminal.Gui.ConsoleDriver)"/>
    ///         <see cref="Init"/> will be called automatically.
    ///     </para>

Reply:
Done. Thanks.